tests/unit/models/test_path.py

```diff
@@ -286,6 +286,34 @@ def test_clear_path(self):
         # pylint: disable=protected-access
         assert DynamicPathManager._clear_path(path) == expected_path
 
+    def test_valid_path_invalid(self):
+        """Test _valid_path for invalid paths."""
+        invalid_a = [
+            "00:00:00:00:00:00:00:02:3",
+            "00:00:00:00:00:00:00:03:2",
+            "00:00:00:00:00:00:00:03:3",
+            "00:00:00:00:00:00:00:04:2"
+        ]
+        assert DynamicPathManager._valid_path(invalid_a) is False
```

Comment:
@Alopalao, these multiple assertions you can turn them into a fixture, that's usually the canonical way with pytest, we indeed do have sometimes multiple subcases for a given test doing multiple assertions but these tend to be when we didn't use to have full pytest support. Since this a new test case you're adding and `pytest.mark.parametrize` should work well here, then let's go for it, OK?

Finally, let's also include cases when `_valid_path` is expected to be True.

Reply:
Added more unit tests for valid and invalid paths. Also checked on the ops team and they reported there is not invalid path deployed.